pkg/log/options.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	defaultOutputLevel        = InfoLevel
+	defaultOutputLevel        = WarnLevel
```

Comment:
We can't change this for the product, the intent is for Info to be the default and changing this now will remove a considerable amount of log info we depend on.

Reply:
So I guess until we have a better solution in #2400, we won't be able to have a simple fix for this #3835, any ideas or should I close this?